Ethereum/ethereum-chainlink/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "declaration": true,
+    "importHelpers": true,
+    "resolveJsonModule": true,
+    "module": "commonjs",
+    "outDir": "dist",
+    "rootDir": "src",
+    "target": "es2017"
```

Comment:
its good practice to enable `strict: "true"` here. It exposes some other bugs etc

Reply:
This is resolved in https://github.com/subquery/ethereum-subql-starter/pull/39/commits/a892fd39b2a620b1465afd667a87ce0b2fe4fb16